packages/react/src/radio-group/root/RadioGroupRoot.tsx

```diff
@@ -33,6 +34,9 @@ const RadioGroupRoot = React.forwardRef(function RadioGroupRoot(
     ...otherProps
   } = props;
 
+  const directionContext = useDirectionContext();
+  const direction = directionContext?.direction ?? 'ltr';
```

Comment:
Since it only has one value, we could simplify this by defaulting `ltr` inside `useDirectionContext()` and returning the value directly, `const direction = useDirectionContext()`